client/src/App.tsx

```diff
@@ -1,14 +1,18 @@
 import { Routes, Route } from "react-router-dom";
+import SettingsBar from "./components/settingsbar/SettingsBar";
 import LogInPage from "./pages/Auth/LogInPage";
 import RegisterPage from "./pages/Auth/RegisterPage";
 import ConversationPage from "./pages/conversation/ConversationPage";
 import GetStartedPage from "./pages/GetStartedPage";
+import SettingsPage from "./pages/SettingsPage/SettingsPage";
 import PageNotFound from "./pages/_PageNotFound";
 
 function App() {
   return (
       <Routes>
         <Route path="/" element={<GetStartedPage />} />
+        <Route path="/settings" element={<SettingsPage />} />
+        <Route path="/setting" element={<SettingsBar />} />
```

Comment:
Pagol achis naki? 

Ekbar tp `/settings` a `<SettingsPage />` lagiechis to settings bar ta alada route a lagiechis kno??????

Ota to component ota to `SettingsPage` a impprt hobe ota nije page to noi ekta 

@srijibmondal 😑😑😑😑😑

Reply:
> Pagol achis naki? 
> 
> Ekbar tp `/settings` a `<SettingsPage />` lagiechis to settings bar ta alada route a lagiechis kno??????
> 
> Ota to component ota to `SettingsPage` a impprt hobe ota nije page to noi ekta 
> 
> @srijibmondal 😑😑😑😑😑